localstack/services/events/events_starter.py

```diff
@@ -1,29 +1,80 @@
 import json
-from moto.events.responses import EventsHandler
+import uuid
+
+from moto.events.models import Rule as rule_model
+from moto.events.responses import EventsHandler as events_handler
+
 from localstack import config
-from localstack.constants import DEFAULT_PORT_EVENTS_BACKEND
+from localstack.constants import (
+    APPLICATION_AMZ_JSON_1_1, DEFAULT_PORT_EVENTS_BACKEND, TEST_AWS_ACCOUNT_ID
+)
 from localstack.services.infra import start_moto_server
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import short_uid
 
 
-def start_events(port=None, asynchronous=None, update_listener=None):
-    port = port or config.PORT_EVENTS
-    backend_port = DEFAULT_PORT_EVENTS_BACKEND
+DEFAULT_EVENT_BUS_NAME = 'default'
 
-    apply_patches()
+# Event rules storage
+EVENT_RULES = {
+    DEFAULT_EVENT_BUS_NAME: []
+}
 
-    return start_moto_server(
-        key='events',
-        port=port,
-        name='Cloudwatch Events',
-        asynchronous=asynchronous,
-        backend_port=backend_port,
-        update_listener=update_listener
-    )
+
+def send_event_to_sqs(event, arn):
+    queue_url = aws_stack.get_sqs_queue_url(arn)
+    sqs_client = aws_stack.connect_to_service('sqs')
+
+    sqs_client.send_message(QueueUrl=queue_url, MessageBody=event['Detail'])
+
+
+def process_events(event, targets):
+    for target in targets:
+        arn = target['Arn']
+        service = arn.split(':')[2]
+
+        if service == 'sqs':
+            send_event_to_sqs(event, arn)
+
+        # TODO implement for more target type: lambda,sns,...
 
 
 def apply_patches():
-    # Patch _put_targets  #2101 Events put-targets does not respond
-    def EventsHandler_put_targets(self):
+    # Fix events arn
+    def rule_model_generate_arn(self, name):
+        return 'arn:aws:events:{region_name}:{account_id}:rule/{name}'.format(
+            region_name=self.region_name, account_id=TEST_AWS_ACCOUNT_ID, name=name
+        )
+
+    events_handler_put_rule_orig = events_handler.put_rule
+
+    def events_handler_put_rule(self):
+        name = self._get_param('Name')
+        event_bus = self._get_param('EventBusName')
+        if not event_bus:
```

Comment:
nit: We could remove lines 54-55 and change line 53 to:
```
event_bus = self._get_param('EventBusName') or DEFAULT_EVENT_BUS_NAME
```

Also, let's assume that `event_bus` is `None` - then line 60 (`EVENT_RULES[event_bus].append(name)`) would insert an item at dict index `None`, right? (which is probably not desired)